src/runtime/components/data/Table.vue

```diff
@@ -4,7 +4,7 @@
       <thead :class="ui.thead">
         <tr :class="ui.tr.base">
           <th v-if="modelValue" scope="col" :class="ui.checkbox.padding">
-            <UCheckbox :model-value="indeterminate || selected.length === rows.length" :indeterminate="indeterminate" aria-label="Select all" @change="onChange" />
+            <UCheckbox :model-value="indeterminate || selected.length === rows.length" :indeterminate="indeterminate" :color="ui.default.checkbox.color" aria-label="Select all" @change="onChange" />
```

Comment:
I think it could be nice to `v-bind="ui.default.checkbox"` here so we can change other props than color.

Reply:
This reminds me of this PR: https://github.com/nuxt/ui/pull/1269